core/chainlink.Dockerfile

```diff
@@ -66,12 +66,15 @@ RUN make chainlink-build
 FROM ubuntu:18.04
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y ca-certificates
+RUN apt-get update && apt-get install -y ca-certificates curl
 
 WORKDIR /root
 
 COPY --from=1 /go/bin/chainlink /usr/local/bin/
 
 EXPOSE 6688
 ENTRYPOINT ["chainlink"]
+
+HEALTHCHECK CMD curl -f http://localhost:6688 || exit 1
```

Comment:
Would the healthcheck be able to run on a different port to the UI? There's an issue with using the 6688 port as it's only opened when the DB lock is grabbed and if you're running a standby node that waits indefinitely for the lock, it'd never be marked as healthy.

Reply:
Yes, that would be possible, I just don't have the Go skills for that. :sweat_smile: 
Would you like to do that or point out a developer to continue the PR?